apps/challenges/aws_utils.py

```diff
@@ -664,6 +664,7 @@ def scale_resources(challenge, worker_cpu_cores, worker_memory):
             challenge.task_def_arn = None
             challenge.save()
         else:
+            response["Error"] = "Scaling worker not supported for inactive worker. Full error:\n" + response["Error"]
```

Comment:
Change message to `Scaling inactive workers not supported`

Reply:
Done